lib/security_blog.js

```diff
@@ -205,46 +206,24 @@ export default class SecurityBlog {
     const reports = content.reports;
     let template = '';
     for (const report of reports) {
-      let cveId = report.cve_ids?.join(', ');
+      const cveId = report.cveIds?.join(', ');
       if (!cveId) {
-        // ask for the CVE ID
-        // it should have been created with the step `--request-cve`
-        cveId = await this.cli.prompt(`What is the CVE ID for vulnerability https://hackerone.com/reports/${report.id} ${report.title}?`, {
-          questionType: 'input',
-          defaultAnswer: 'TBD'
-        });
-        report.cve_ids = [cveId];
-        content[kChanged] = true;
+        this.cli.error(`CVE ID for vulnerability https://hackerone.com/reports/${report.id} not found`);
```

Comment:
missing report title, maybe you can create the string `for vulnerability https://hackerone.com/reports/${report.id} ${report.title}.` once